packages/split-view/test/split-view.test.ts

```diff
@@ -972,6 +972,24 @@ describe('SplitView', () => {
         expect(testPanel).to.be.null;
     });
 
+    it('allows a custom label when resizable if specified', async () => {
```

Comment:
is there an existing test that falls back to the default string if custom label is not specified? If not, can you add one please?

Reply:
Added one!